micro/core/gunicornapp.py

```diff
@@ -26,3 +28,21 @@ def load_config(self):
 
     def load(self):
         return self.__app
+
+    def run(self):
+        if self.cfg.check_config:
+            try:
+                self.load()
+            except Exception:
+                msg = "Error while loading the application"
+                sys.exit(msg)
+
+            sys.exit(0)
+
+        if self.cfg.spew:
+            debug.spew()
```

Comment:
Is this to allow some kind of debug session?

Reply:
yes, it's `gunicorn` description is:
```desc = """\
        Install a trace function that spews every line executed by the server.
        This is the nuclear option.```